bin/secrets_manager

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env php
+<?php
+if (file_exists(__DIR__.'/vendor/autoload.php')) {
+    $autoloaderPath = __DIR__.'/vendor/autoload.php';
+} elseif (file_exists(__DIR__.'/../vendor/autoload.php')) {
+    $autoloaderPath = __DIR__ . '/../vendor/autoload.php';
+} elseif (file_exists(__DIR__.'/../../../autoload.php')) {
+    $autoloaderPath = __DIR__ . '/../../../autoload.php';
+} else {
+    die('Could not find autoload.php');
+}
+$classLoader = require $autoloaderPath;
+
+$getOpt = new \GetOpt\GetOpt([
+    ['s', 'secrets_manager_class', \GetOpt\GetOpt::REQUIRED_ARGUMENT, 'Set the SecretsManager class (without namespace).'],
+    ['p', 'secrets_provider_class', \GetOpt\GetOpt::REQUIRED_ARGUMENT, 'Set the SecretsProvider class (without namespace).'],
+], [\GetOpt\GetOpt::SETTING_STRICT_OPERANDS => true]);
+
+$getOpt->addCommands([\GetOpt\Command::create('printenv', 'secrets_manager_printenv')]);
+$getOpt->addCommands([\GetOpt\Command::create('checkvalues', 'secrets_manager_checkvalues')]);
+
+$getOpt->process();
+$options = $getOpt->getOptions();
+$operands = $getOpt->getOperands();
+$command = $getOpt->getCommand();
+
+// show help and quit
+if (!$command || $getOpt->getOption('help')) {
+    echo $getOpt->getHelpText();
+    exit;
+}
+
+if (isset($options['secrets_manager_class'])) {
+    $project_settings = new Kducharm\ProjectSettings\ProjectSettings();
+    $secrets_class = 'Kducharm\ProjectSettings\\' . $options['secrets_manager_class'];
```

Comment:
Are there supposed to be 2 backslashes here?

Reply:
Yes otherwise it thinks it's an escaping a single quote